### DISCUSSION TITLE
Is the `android-actions/setup-android@v2` action used in `AndroidCIWithGmd.yaml` official?

### DISCUSSION THREAD

**Discussion by GSala** [android/nowinandroid#610] [General]
Hi, 

I was getting some inspiration from this project around a Github Actions CI setup for an Android project, and I noticed that `android-actions/setup-android@v2` was being used to set up the SDK. 

It doesn't look like this action is officially maintained by Google or the Android team. Can someone confirm? 
Also, if it isn't, is it the best idea to use this action here, in a repo that many will use as a go-to reference?

**Comment by dturner** [comment]
We've had a lot of issues with flaky CI so we're just testing this out at the moment. Not an official recommendation until we know it's reliable, which it definitely isn't at present.

**Comment by GSala** [comment]
Just coming back here to write some things that the Google Play/Android Tools teams could do to help the community more in terms of DevOps:
- An official Github Action (and other CI alternatives) to publish an app to the store
- An official Docker (or container) image to build or run tests in
- Easier API access to the Google Play (without having to manage Google Cloud console permissions 😩)

Honest feedback. It seems to me that these things are a drop in the bucket compared to the amount of work that goes into Android Studio, for example.
